File: Services/CategoryServices/DrivingSchoolApp.Services.CategoryServices.DCategoryAPI/DataAccess/Entities/WorkExperience.cs
using DrivingSchoolApp.Services.CategoryServices.DCategoryAPI.DataAccess.Entities.Enums;

namespace DrivingSchoolApp.Services.CategoryServices.DCategoryAPI.DataAccess.Entities;

public class WorkExperience : Base
{
	public int Experience { get; set; }
	public string Company { get; set; }
	public TypeOfWorkExperience TypeOfWorkExperience { get; set; }
	public int? InstructorId { get; set; }
	public virtual Instructor Instructor { get; set; }
}

File: Frontend/DrivingSchoolWeb/src/app/categories/category-c/models/work-experience.ts
import {Base} from 'src/app/categories/shared/models/base';
import {TypeOfWorkExperience} from 'src/app/categories/category-c/models/enums/work-experience-type';

export interface WorkExperience extends Base{
  experience:number,
  company:string,
  categoryType:TypeOfWorkExperience
}
